soc/arm/nxp_imx/rt/Kconfig.soc

```diff
@@ -319,8 +319,9 @@ config IPG_DIV
 	int "IPG clock divider"
 	range 0 3
 
-menuconfig NXP_IMX_RT_BOOT_HEADER
+config NXP_IMX_RT_BOOT_HEADER
 	bool "Enable the boot header"
+	default y
```

Comment:
Instead of changing the default here, please add `select NXP_IMX_RT_BOOT_HEADER` to `config CODE_SEMC`, similar to the other choices that need it (i.e., `CODE_FLEXSPI`)

Reply:
Thanks for your review, I fixed it